embedded-graphics/src/lib.rs

```diff
@@ -182,7 +185,7 @@ use crate::pixelcolor::PixelColor;
 /// Here's an example for an imaginary display that has a 64x64px framebuffer of 8 bit values that
 /// communicates over a (simplified) SPI interface:
 ///
-/// ```rust
+/// ```ignore
```

Comment:
Why does this test fail? Is it in-scope for this PR to fix it?

Reply:
This was disabled because there was no 8 bit color type when I started development, but it is fixed now.